### PR TITLE
Poké: rework display active subscription/plan

### PR DESCRIPTION
**Pull request by PopDaph**
## Description

I know this is more code but Poké deserves some love 🙈 
This PR is about removing the JSON viewer to replace by a table -> splitting in both the Subscription info with the product limitations from the plan. 

Screenshot 2024-04-04 at 18 14 15

Next step is to move the buttons to upgrade/downgrade a workspace and add a new one for Enterprise onboarding; 

## Risk

/

## Deploy Plan

/

### CODE REVIEW

File: front/pages/poke/[wId]/index.tsx [hunk 1]
```diff
@@ -329,9 +328,215 @@ const WorkspacePage = ({
                   </DropdownMenu.Items>
                 </DropdownMenu>
               </div>
+              <div className="border-material-200 my-4 flex flex-col rounded-lg border p-4">
```
Comment (flvndvd) [inline front/pages/poke/[wId]/index.tsx:331]: 😱 Can you move this to a dedicated table like we did for other tables?

**Review by fontanierh — APPROVED** [review]
Looks great !

File: front/pages/poke/[wId]/index.tsx (hunk 1 above)
Comment (PopDaph) [reply front/pages/poke/[wId]/index.tsx:331]: Yes @flvndvd 😄

**Comment by PopDaph** [comment]
As discussed IRL I also moved the order of tables and renamed the title for the table with history of subscriptions.

File: front/components/poke/subscriptions/table.tsx [hunk 2]
```diff
@@ -31,11 +49,190 @@ export function SubscriptionsDataTable({
...
 }
+
+export function ActiveSubscriptionTable({
+  subscription,
+}: {
+  subscription: SubscriptionType;
+}) {
+  const activePlan = subscription.plan;
+  return (
+    <div className="flex flex-col space-y-8 pt-4">
```
Comment (flvndvd) [inline front/components/poke/subscriptions/table.tsx:68]: Any chance we could factorize this? Like having an object like we do for `data_table`?

Comment (PopDaph) [reply front/components/poke/subscriptions/table.tsx:68]: Yes! Merging as is so I can focus on the new Enterprise form this morning, I'll open a new PR to handle this feedback right after 🙇🏻‍♀️

**Pull request merged**